src/status_im/chat/models/images.cljs

```diff
@@ -159,7 +159,12 @@
                                                    [:node :image :uri])]
                                    (swap! albums update-in [1 :data] conj (merge album {:uri uri}))
```

Comment:
why not sort it here ?

Reply:
I am sorting before dispatching, after fetching all the albums data.